manifests/bootstrap_rpm.pp

```diff
@@ -19,8 +19,7 @@
 
   include trusted_ca
   trusted_ca::ca { 'katello_server-host-cert':
```

Comment:
I think it's possible to still have them with a resource collector. From https://www.puppet.com/docs/puppet/8/lang_collectors.html
```puppet
  File <| title == $server_ca_cert |> ->
  trusted_ca::ca { 'katello_server-host-cert':
```

Reply:
The way puppet-certs is designed and works, the current setup will never have a File resource. When the tarfile is extracted on an external foreman-proxy it just lays it out on disk and skips generate. On Foreman, the only place generate is true, the Puppet custom resources that are used do not auto-generate a File resource. So there is currently no setup where a File resource for these will exist.

Reply:
Can we then at least change `class { 'foreman_proxy_content::bootstrap_rpm':` to add `require => Class['certs']` so we know the ordering is correct?